### ISSUE TITLE
Confusing behavior of escape key

### ISSUE THREAD

**Issue by jwyang-qraft** [microsoft/vscode#135786]
I am enjoying the experience of editing jupyter notebook on vscode!
However, I noticed one subtle difference in interaction.

There are 3 modes in vscode-jupyter: unselected, command, edit.
If I press escape key in edit mode, I move to command mode.
If I am in command mode, I move to unselected mode.
This dual behavior is quite frustrating, because I have to make sure
that I press escape key only once.
In jupyter-lab pressing escape key in command mode does not change the mode I am in.
I think this might be better default operation.
Also, it will be great if there is way to disable unselected mode at least.
(Correct me if this feature is already  available)

**Comment by johapark** [comment]
I've had the same frustration and found the solution here: 
https://github.com/microsoft/vscode/issues/121129#issuecomment-876457460

**Comment by roblourens** [comment]
It sounds like this is a duplicate of https://github.com/microsoft/vscode/issues/121129
